### PR TITLE
Minor auth sidecar cleanup

### PR DESCRIPTION
**Pull request by jedcunningham**
A few renames and comment changes were missed in the original PR (#1103).

### CODE REVIEW

File: charts/grafana/templates/grafana-auth-sidecar-configmap.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ################################
```
Comment (danielhoherd) [inline charts/grafana/templates/grafana-auth-sidecar-configmap.yaml:1]: Nitpick, but while we're cleaning up these three comments, can we make the hash lines consistent?
```suggestion
#################################
```

File: charts/kibana/templates/kibana-auth-sidecar-configmap.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ################################
```
Comment (danielhoherd) [inline charts/kibana/templates/kibana-auth-sidecar-configmap.yaml:1]: Nitpick, but while we're cleaning up these three comments, can we make the hash lines consistent?
```suggestion
#################################
```

File: charts/prometheus/templates/prometheus-auth-sidecar-configmap.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ################################
```
Comment (danielhoherd) [inline charts/prometheus/templates/prometheus-auth-sidecar-configmap.yaml:1]: Nitpick, but while we're cleaning up these three comments, can we make the hash lines consistent?
```suggestion
#################################
```

**Review by danielhoherd — APPROVED** [review]

File: charts/grafana/templates/grafana-auth-sidecar-configmap.yaml (hunk 1 above)
Comment (jedcunningham) [reply charts/grafana/templates/grafana-auth-sidecar-configmap.yaml]: Works for me. They were consistently inconsistent everywhere so I left them 🤷‍♂️

**Pull request merged**